pkg/cluster/bootstrap.go

```diff
@@ -33,7 +35,11 @@ func (c *Cluster) Bootstrap(ctx context.Context) error {
 			return err
 		}
 	}
-
+	if c.config.ClusterResetRestorePath != "" && c.config.RKE1Snapshot {
```

Comment:
Can you include a check where restore path is empty and rke1 snapshot is true and have it error?